R/execute.R

```diff
@@ -73,7 +113,7 @@ rmf_run_modflow.modflow <- function(modflow,
   rmf_write_nam(modflow$nam, file = 'input.nam')
```

Comment:
Can this (lines 113-123) not be replaced by `rmf_write` which writes the entire `modflow` object?

Reply:
I suppose so, but I propose we work on that wherever we work on the top-level functionality. Will add a todo comment now.